cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```
@@ -212,9 +212,8 @@ def create_cephCSIKeyring_cephFSProvisioner(self):
         ret_val, json_out, err_msg = self._common_cmd_json_gen(cmd_json)
         # if there is an unsuccessful attempt,
         if ret_val != 0 or len(json_out) == 0:
-            raise ExecutionFailureException(
-                "'auth get-or-create client.csi-cephfs-provisioner' command failed.\n" +
-                "Error: {}".format(err_msg if ret_val != 0 else self.EMPTY_OUTPUT_LIST))
+            # silently return an empty string
+            return ''
```

We still need to raise an error if we fail to create the key...